Source/Core/DolphinQt/Settings.h

```diff
@@ -54,6 +54,10 @@ class Settings final : public QObject
   void SetUserStylesEnabled(bool enabled);
   bool AreUserStylesEnabled() const;
 
+  void GetToolTipStyle(QColor& window_color, QColor& text_color, QColor& emphasis_text_color,
```

Comment:
This could be have static for now but in the future it might depend on other settings.